projects/deno.land/package.yml

```diff
@@ -19,11 +19,17 @@ runtime:
     DENORT_BIN: '{{prefix}}/bin/denort'
 
 dependencies:
-  x86-64:
+  darwin/x86-64:
     llvm.org: 17 # libunwind
 
 build:
   script:
+    - run: |
+        set -o allexport
+        source <(pkgx +llvm.org@17)
+        set +o allexport
+      if: darwin/aarch64
```

Comment:
so, this should be a build-time dependency on llvm.org^17 for darwin, right? since we're introducing it here, and run-time for x86-64?

Reply:
I'm still doing some tests, but I had to step away from the computer. Will resume later today.

Reply:
no rush; i always appreciate more eyes on this stuff. it's complicated, and it's more complicated when compiling compilers.

Reply:
> so, this should be a build-time dependency on llvm.org^17 for darwin, right?

Building on darwin+aarch64 needs llvm otherwise: https://github.com/pkgxdev/pantry/actions/runs/8492104954/job/23264754385?pr=5751#step:6:921

> and run-time for x86-64?

Running on darwin+x86-64 needs llvm otherwise: https://github.com/pkgxdev/pantry/actions/runs/8492104954/job/23264754342?pr=5751#step:6:36

My theory however is that the deno used by brewkit only needs llvm in runtime for darwin+x86-64 because it had llvm set during build time.

Perhaps after this PR, deno will be rebuilt and it will no longer need llvm as runtime dependency on darwin+x86-64, but that's just a guess.

Reply:
It appears that deno will link in libunwind on x86-64 if present. So, yes, perhaps excluding it from build will solve it.